Core/src/main/java/de/Ste3et_C0st/ProtectionLib/main/plugins/fGriefdefenderAPI.java

```diff
@@ -9,20 +9,32 @@
 import org.bukkit.event.EventHandler;
 import org.bukkit.event.Listener;
 import org.bukkit.plugin.Plugin;
+
 import com.flowpowered.math.vector.Vector3i;
 import com.griefdefender.api.GriefDefender;
 import com.griefdefender.api.claim.Claim;
 import com.griefdefender.api.claim.ClaimManager;
+import com.griefdefender.api.Subject;
+import com.griefdefender.api.Tristate;
+import com.griefdefender.api.permission.flag.Flag;
 
 import de.Ste3et_C0st.ProtectionLib.events.RegionClearEvent;
 import de.Ste3et_C0st.ProtectionLib.main.ProtectionConfig;
 import de.Ste3et_C0st.ProtectionLib.main.ProtectionLib;
 
 public class fGriefdefenderAPI extends ProtectionConfig implements Listener {
+	private final Flag PROTECTIONLIB_BUILD_PROTECT;
 
 	public fGriefdefenderAPI(Plugin plugin) {
 		super(plugin);
 		Bukkit.getPluginManager().registerEvents(this, ProtectionLib.getInstance());
+
+        PROTECTIONLIB_BUILD_PROTECT = Flag.builder()

```

Comment:
Why is this custom flag needed? This plugin can just make use of `Flags.BLOCK_PLACE` assuming the only check is doing is on block place.